eclipse-scout-core/src/calendar/ResourcePanel.ts

```diff
@@ -41,29 +44,60 @@ export class ResourcePanel extends Widget {
 class ResourcePanelTreeBox extends TreeBox<string> {
   declare tree: ResourcePanelTree;
   declare lookupCall: CalendarResourceLookupCall;
+  tooltipSupport: StaticTooltip;
 
   protected override _render() {
     super._render();
     this.removeMandatoryIndicator();
+    this._installTooltipSupport();
+  }
+
+  protected override _remove() {
+    super._remove();
+    this.tooltipSupport.close();
   }
 
   protected override _renderFocused() {
     // NOP
   }
 
+  protected _installTooltipSupport() {
+    let model = {
+      parent: this,
+      text: '${textKey:ui.AtLeastOneCalendarHasToBeVisible}'
+    };
+    this.tooltipSupport = new StaticTooltip(model);
+  }
+
   protected override _createNode(lookupRow: LookupRow<string>): TreeBoxTreeNode<string> {
     let node = super._createNode(lookupRow);
     node.expanded = true;
     return node;
   }
 
   protected override _onTreeNodesChecked(event: TreeNodesCheckedEvent) {
+    this._closeTooltip();
     // Make impossible to uncheck all nodes
     if (arrays.hasElements(this.tree.checkedNodes)) {
       super._onTreeNodesChecked(event);
     } else if (!this._populating) {
       // Reapply the value to the tree
       this._syncValueToTree(this.value);
+      this._createImpossibleToUncheckTooltip(event.nodes[0]);
+    }
+  }
+
+  protected _createImpossibleToUncheckTooltip(node: TreeNode) {
```

Comment:
I wouldn't use the concept of TooltipSupport here, instead create Tooltip directly, see for example:
StatusIndicator._showTooltip

I wouldn't even use a tooltip here but a popup instead? I tried this in the GUI and the tooltip feels unusual to me. When a calendar name is too long, I can get two overlapping tooltips.